build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -24,7 +24,9 @@ CONFIG_TOOLS_DIR="../tools"
 CONFIG_APP_BINARY_SEPARATION=y
 CONFIG_NUM_APPS=5
 # CONFIG_APPS_RAM_REGION_SHAREABLE is not set
-# CONFIG_SUPPORT_COMMON_BINARY is not set
+CONFIG_SUPPORT_COMMON_BINARY=y
```

Comment:
This should be disabled. or Need to modify flash partition map.

Reply:
Thanks, I changed CONFIG_SUPPORT_COMMON_BINARY to disable.